build.sbt

```diff
@@ -41,14 +40,7 @@ libraryDependencies ++= Seq(
   "com.typesafe.scala-logging" %% "scala-logging" % "3.7.1",
   "com.typesafe.slick" %% "slick" % "3.2.0",
   "com.zaxxer" % "HikariCP" % "2.3.9",
-  // TODO[GAWB-3052] remove
-  "io.spray" %% "spray-can" % sprayV,
-  // TODO[GAWB-3052] remove
-  "io.spray" %% "spray-client" % sprayV,
-  // TODO[GAWB-3052] remove if possible -- this may not be possible or desirable
-  "io.spray" %% "spray-json" % "1.3.3", // NB: Not at sprayV. 1.3.4 does not exist.
-  // TODO[GAWB-3052] remove
-  "io.spray" %% "spray-routing-shapeless23" % sprayV,
+  "io.spray" %% "spray-json" % "1.3.3",
```

Comment:
We may want to use
```
"com.typesafe.akka"   %%  "akka-http-spray-json"
```
instead, which is what Rawls does.

(Although plain old `spray-json` does [appear to be maintained](https://github.com/spray/spray-json/commits/master) for now Scala versions.)

Reply:
We are on line 36. I'll double check this to be sure if we need both.

Reply:
Good call ... it's totally not necessary to include this.